src/main/java/com/uber/jenkins/phabricator/conduit/Differential.java

```diff
@@ -49,6 +49,14 @@ public Differential(JSONObject rawJSON) {
         this.rawJSON = rawJSON;
     }
 
+    public String getDIffID() {
```

Comment:
@jjx It may be inconsistent with the surrounding code like `getRevisionID` though.